### ISSUE TITLE
Accessibility: Required ARIA attributes must be provided

### ISSUE THREAD

**Issue by bedmison** [web-cat/web-cat-subsystem-Core#9]
Some controls are missing the required ARIA attributes. The required ARIA attributes that are not present: aria-expanded, aria-controls. See [more information Link opens in a new window](https://dequeuniversity.com/rules/axe/4.10/aria-required-attr?application=AxeChrome)

This manifests in the format select box under the TA comments box.
